src/openrct2/rct12/RCT12.h

```diff
@@ -320,54 +320,54 @@ namespace OpenRCT2::RCT12
     };
 } // namespace OpenRCT2::RCT12
 
-enum
-{
-    MAP_ELEM_TRACK_SEQUENCE_GREEN_LIGHT = (1 << 7),
-};
-
-constexpr uint8_t RCT2_STRING_FORMAT_ARG_START = 123;
-constexpr uint8_t RCT2_STRING_FORMAT_ARG_END = 141;
-constexpr uint8_t RCT2_STRING_FORMAT_COLOUR_START = 142;
-constexpr uint8_t RCT2_STRING_FORMAT_COLOUR_END = 156;
+constexpr uint8_t kRCT12StringFormatArgStart = 123;
+constexpr uint8_t kRCT12StringFormatArgEnd = 141;
+constexpr uint8_t kRCT12StringFormatColourStart = 142;
+constexpr uint8_t kRCT12StringFormatColourEnd = 156;
 
-constexpr uint8_t RCT12_SOUND_ID_NULL = 0xFF;
+constexpr uint8_t kRCT12SoundIdNull = 0xFF;
 
 using RCT12RideId = uint8_t;
-constexpr RCT12RideId RCT12_RIDE_ID_NULL = 255;
+constexpr RCT12RideId kRCT12RideIdNull = 255;
 
-constexpr uint8_t RCT12_BANNER_INDEX_NULL = std::numeric_limits<uint8_t>::max();
+constexpr uint8_t kRCT12BannerIndexNull = std::numeric_limits<uint8_t>::max();
 
-constexpr uint8_t RCT12_TILE_ELEMENT_SURFACE_EDGE_STYLE_MASK = 0xE0;   // in RCT12TileElement.properties.surface.slope
-constexpr uint8_t RCT12_TILE_ELEMENT_SURFACE_WATER_HEIGHT_MASK = 0x1F; // in RCT12TileElement.properties.surface.terrain
-constexpr uint8_t RCT12_TILE_ELEMENT_SURFACE_TERRAIN_MASK = 0xE0;      // in RCT12TileElement.properties.surface.terrain
+constexpr uint8_t kRCT12SurfaceElementEdgeStyleMask = 0xE0;         // in RCT12TileElement.properties.surface.slope
+constexpr uint8_t kRCT12SurfaceElementWaterHeightMask = 0x1F;       // in RCT12TileElement.properties.surface.terrain
+constexpr uint8_t kRCT12SurfaceElementTerrainMask = 0xE0;           // in RCT12TileElement.properties.surface.terrain
```

Comment:
Given these are masks, should we write them as binary, too?

```
constexpr uint8_t kRCT12SurfaceElementEdgeStyleMask = 0b11100000;         // in RCT12TileElement.properties.surface.slope
constexpr uint8_t kRCT12SurfaceElementWaterHeightMask = 0b11111;       // in RCT12TileElement.properties.surface.terrain
constexpr uint8_t kRCT12SurfaceElementTerrainMask = 0b11100000;
```

(Sorry, I can't seem to make this a proper suggestion right now.)

Reply:
Is it okay if I do this in another PR? There are loads more tile element constants in this file and I want to refactor them all (and move the RCT12 tile element stuff to a file of its own).

Reply:
Sure, no biggie.